apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -77,38 +75,6 @@ function add_public_coming_soon_to_settings_endpoint_post( $input, $unfiltered_i
 }
 add_filter( 'rest_api_update_site_settings', __NAMESPACE__ . '\add_public_coming_soon_to_settings_endpoint_post', 10, 2 );
 
-/**
- * This filter adds the coming soon page to the list of page templates using the `vertical_templates` hook
- *
- * @param array $templates The array of all available page templates.
- *
- * @return array The updated $templates array
- */
-function add_public_coming_soon_page_template( $templates ) : array {
```

Comment:
Should we also get rid of `get_coming_soon_page_template()` as well?

Reply:
yes, that function was replaced with the COMING_SOON_PAGE_TEMPLATE constant which has also been removed by deleting `coming-soon-page-template.php`  in this PR 👍